dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp

```diff
@@ -163,6 +200,328 @@ PrehandleResult KVStore::preHandleSnapshotToFiles(
     return result;
 }
 
+static inline std::vector<std::string> getSplitKey(
+    LoggerPtr log,
+    KVStore * kvstore,
+    RegionPtr new_region,
+    std::shared_ptr<DM::SSTFilesToBlockInputStream> sst_stream)
+{
+    // We don't use this is the single snapshot is small, due to overhead in decoding.
+    // TODO(split) find solution if the snapshot has too many untrimmed data.
+    // TODO(split) recover this after integration test passes.
+    // constexpr size_t PARALLEL_PREHANDLE_THRESHOLD = 1 * 1024 * 1024 * 1024;
+    constexpr size_t PARALLEL_PREHANDLE_THRESHOLD = 1 * 1024 * 1024;
```

Comment:
does it pass?

Reply:
yes, it passed the ut and manual test. however, i'd like to leave this until performance test is finished